nameko_twilio.py

```diff
@@ -5,11 +5,13 @@
 class Twilio(DependencyProvider):
 
     def setup(self):
-        self.account_sid = self.container.config['account_sid']
-        self.auth_token = self.container.config['auth_token']
+        self.twilio_account_sid = self.container.config['account_sid']
+        self.twilio_auth_token = self.container.config['auth_token']
```

Comment:
There's some confusion here. Look back at my previous comemnt on this. I said that the config variables should be renamed.
`self.account_sid` and `self.auth_token` are fine, because they belong to the `Twilio` class. However, the ones in the config are named to generically.